modules/tiles/src/tileset/helpers/bounding-volume.js

```diff
@@ -69,17 +69,41 @@ export function createBoundingVolume(boundingVolumeHeader, transform, result) {
 }
 
 function createBox(box, transform, result) {
+  // https://math.gl/modules/culling/docs/api-reference/oriented-bounding-box
+  // 1. A half-axes based representation.
   // box: An array of 12 numbers that define an oriented bounding box.
   // The first three elements define the x, y, and z values for the center of the box.
   // The next three elements (with indices 3, 4, and 5) define the x axis direction and half-length.
   // The next three elements (indices 6, 7, and 8) define the y axis direction and half-length.
   // The last three elements (indices 9, 10, and 11) define the z axis direction and half-length.
+  // 2. A half-size-quaternion based representation.
+  // box: An array of 10 numbers that define an oriented bounding box.
+  // The first three elements define the x, y, and z values for the center of the box in a right-handed 3-axis (x, y, z) Cartesian coordinate system where the z-axis is up.
+  // The next three elements (with indices 3, 4, and 5) define the halfSize.
+  // The last four elements (indices 6, 7, 8 and 10) define the quaternion.
   const center = new Vector3(box[0], box[1], box[2]);
   transform.transform(center, center);
-
-  const xAxis = transform.transformAsVector(box.slice(3, 6));
-  const yAxis = transform.transformAsVector(box.slice(6, 9));
-  const zAxis = transform.transformAsVector(box.slice(9, 12));
+  let origin = [];
```

Comment:
For another PR: I wonder if we can just convert the box into a math.gl OBB and use OBB methods to do these tranformations, rather than add so much logic here.

Maybe we need to add another method in math.gl, that would be OK.

Generally I like using other libraries to minimize complexity here, if the code can be generalized.

But of course, first priority is to fix the issue.